functions/src/api/v1/rooms/createRoom.ts

```diff
@@ -101,9 +144,66 @@ export const createRoom = async ({
     await RoomDao.update({
         id: roomId,
         sid: roomSid,
+        name: name || roomId,
     })
+
+    let processDestinationsResults = {}
+    if (hostName && destinations) {
+        processDestinationsResults = await processDestinations(
+            roomId,
+            userId,
+            hostName,
+            destinations,
+            sendsAt
+        )
+    }
+
     return {
         roomId,
         roomSid,
+        ...processDestinationsResults,
     }
 }
+
+const processDestinations = async (
+    roomId: RoomId,
+    userId: UID,
+    hostName: string,
+    destinations: string,
+    sendsAt?: string
+): Promise<ProcessDestinationsResponse> => {
+    if (sendsAt) {
+        const sendsAtValues = parseSendsAt(sendsAt)
+        const reminderIds: ReminderId[] = []
+        for (const sendAt of sendsAtValues) {
+            const id = await createReminder({
+                roomId,
+                userId,
+                sendAtParameter: sendAt,
+                hostName,
+                destinationsParameter: destinations,
+            })
+            reminderIds.push(id)
+        }
+        return {
```

Comment:
why don't you return directly `reminderIds` ?

Reply:
This is a point we should discuss, since it is a convention used over probably all API responses. In my opinion we should not to wrap the API response with a key, since the return value is already made clear by the `swagger` and the `endpoint` url. This would cause having a double wrapping: on one side we wrap the response on the API, on the other side we'll have to unwrap it on the client side

Reply:
the route name is `createRoom` it can either return reminderIds or email and sms sent. I wanted to assert that in the response.